src/modules/form/components/viewable/DynamicViewFields.tsx

```diff
@@ -74,17 +75,19 @@ const DynamicViewFields: React.FC<Props> = ({
     }
 
     const itemComponent = (
-      <DynamicViewField
-        key={item.linkId}
-        item={item}
-        value={isDisabled ? undefined : values[item.linkId]}
-        nestingLevel={nestingLevel}
-        horizontal={horizontal}
-        pickListRelationId={pickListRelationId}
-        // Needed because there are some enable/disabled and autofill dependencies that depend on PickListOption.labels that are fetched (PriorLivingSituation is an example)
-        adjustValue={itemChanged}
-        {...props}
-      />
+      <Grid item>
```

Comment:
The expectations is that this is always rendered inside DynamicView which provides the `<Grid container>`?

Reply:
Yeah, exactly.